### PR TITLE
Cages GA Attestation

### PR DESCRIPTION
**Pull request by hanneary**
# Why
Need to use new update attestation for GA

# How
- Pull attestation doc from Cage
- Verify pub keys and PCRs from Cage on TLS handshake
- Deprecate existing attestation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,5 +21,11 @@ This project includes a [sample application](https://github.com/evervault/everva
 ## Contributing
 Bug reports and pull requests are welcome on GitHub.
 
+## Cages
+
+Cage attestation is written in Rust and Kotlin bindings can be found [here](https://github.com/evervault/attestation-doc-validation/tree/main/kotlin-attestation-bindings) that are used to intgrate them into this project.
+
+If you wish to update the Rust bindings, make any necessary changes in the Atttesation crate and run the following [script](https://github.com/evervault/attestation-doc-validation/blob/main/kotlin-attestation-bindings/build-libs.sh) to build and copy the shared library files for each architecture into this repo
+
```
Comment (donaltuohy) [inline README.md:29]:
```suggestion
## Cages

Cage attestation is written in Rust and Kotlin bindings can be found [here](https://github.com/evervault/attestation-doc-validation/tree/main/kotlin-attestation-bindings) that are used to integrate them into this project.

If you wish to update the Rust bindings, make any necessary changes in the Attesation crate and run the following [script](https://github.com/evervault/attestation-doc-validation/blob/main/kotlin-attestation-bindings/build-libs.sh) to build and copy the shared library files for each architecture into this repo.

```

File: settings.gradle.kts [hunk 2]
```diff
@@ -22,4 +22,4 @@ include(":sampleapplication")
 include(":evervault-inputs")
 include(":evervault-cages")
 
-//include(":examples:shared")
\ No newline at end of file
+//include(":examples:shared")
```
Comment (donaltuohy) [inline settings.gradle.kts:25]: Was this left in by accident?

**Review by donaltuohy — APPROVED** [review]
lgtm! just two nitpicks

File: settings.gradle.kts (hunk 2 above)
Comment (hanneary) [reply settings.gradle.kts]: think so!

**Pull request merged**